product-service/handlers/getProductsById.js

```diff
@@ -0,0 +1,35 @@
+import products from '../mocks/products.json';
+import { ERRORS, ERROR_TYPES } from '../constants/error';
+import { RESPONSE_STATUSES } from '../constants/response';
+import { getResponse } from '../helpers/response';
+
+const getProduct = (id = '') => new Promise((resolve, reject) => {
+  if (!id) {
+    reject(ERROR_TYPES.NO_ID);
+  }
+
+  const product = products.find((item) => item.id === id);
+
+  if (product) {
+    resolve(product);
+  } else {
+    reject(ERROR_TYPES.NOT_FOUND);
+  }
+});
+
+export const getProductsById = async (event = {}) => {
```

Comment:
getProductById

Reply:
Totally agree, it makes more sense, but according to the task requirements it should be called getProductsById

Reply:
have a look nice example here:
https://github.com/VarvaraZadnepriak/product-service-updated-demo